release.py

```diff
@@ -1,35 +1,327 @@
+#!/usr/bin/env python
+from datetime import datetime
+import uuid
+import os
+from shutil import copyfile
+import stat
 from clingon import clingon
+import codecs
+from future.moves import subprocess
 import git
+from git.exc import GitCommandError
+import requests
+import semver
+import logging
+
+# TODO param this
+RELEASE_BRANCH = 'release'
+
+
+class PullRequest(object):
+    def __init__(self, github_api_response):
+        self.title = github_api_response['title']
+        self.url = github_api_response['html_url']
+        self.head_sha1 = github_api_response['head']['sha']
+        # we consider that if the pr has a merged_at date, it has been merged
+        self.is_merged = github_api_response['merged_at'] is not None
+        self.raw_response = github_api_response
+        logging.debug('pr: {} -- {}'.format(self.title, self.url))
+        self._labels = None
+
+    def fetch_labels(self, auth):
+        """ call github to fetch the labels of the pr """
+        if not self._labels:
+            label_query = self.raw_response['_links']['issue']['href'] + '/labels'
+            self._labels = [r['name'] for r in requests.get(label_query, auth=auth).json()]
+        return self._labels
+
+
+def write_debian_changelog(write_lines, project_path):
+    changelog_path = "debian/changelog"
```

Comment:
you're absolutely right, with it we can be rid of the ugly changelog manipulation.

done,
thanks!